sang-kun/github-profile/github-profile/src/index.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "./index.css";
+import App from "./App";
+import reportWebVitals from "./reportWebVitals";
+import "./fontawesome/css/all.min.css";
```

Comment:
You should not do this way. Let me explain why you should not.
We just need import 'all.min.css', but in our project, we include the whole library fontawesome. 
=> We need to remove it, and try some way to import only all.min.css file OR just import necessary component. 
Please watch out this documentation: https://fontawesome.com/docs/web/setup/packages